### PR TITLE
Add ToS. Closes #1050

### LINKED ISSUE #1050: add ToS

### PR DESCRIPTION
**Pull request by juliangruber**
Closes https://github.com/filecoin-station/desktop/issues/1050

- [x] Confirm with legal whether existing users need to accept update ToS

### CODE REVIEW

File: renderer/src/pages/Onboarding.tsx [hunk 1]
```diff
@@ -19,12 +19,12 @@ const Loading = () => {
...
   const navigate = useNavigate()
   const [isLoading, setIsLoading] = useState<boolean>(true)
-  const [isOnboardingCompleted, setIsOnboardingCompleted] = useState<boolean|null>()
+  const [isOnboardingCompleted, setIsOnboardingCompleted] = useState<boolean|null>(false)
 
   useEffect(() => {
     (async () => {
       await sleep(2000)
-      setIsOnboardingCompleted(await getOnboardingCompleted())
+      // setIsOnboardingCompleted(await getOnboardingCompleted())
```
Comment (juliangruber) [inline renderer/src/pages/Onboarding.tsx:27]: Revert before merge

**Comment by bajtos** [comment]
Nice!

We also discussed that Station should ask existing users to accept the terms of service after they upgrade to the upcoming version. Do you still plan to implement that?

**Comment by juliangruber** [comment]
Good point, let me check with legal

**Comment by juliangruber** [comment]
We don't need to ask existing users again

**Review by patrickwoodhead — APPROVED** [review]

**Review by bajtos — COMMENTED** [review]
👌🏻

**Pull request merged**